lib/cfg-grammar.y

```diff
@@ -1104,16 +1104,16 @@ driver_option
     ;
 
 threaded_dest_driver_option
-	: KW_RETRIES '(' LL_NUMBER ')'
+	: KW_RETRIES '(' positive_integer ')'
```

Comment:
This (deleted) code snippet says explicitly that this config option must be a positive number. If it gets a non-positive number, the default value (3) will be applied with a warning message.